tests/common/helpers/platform_api/component.py

```diff
@@ -0,0 +1,72 @@
+"""
+This component provides an interface to remotely interact with Modules
```

Comment:
s/This component provides an interface to remotely interact with Modules/This module provides an interface to remotely interact with Components/